### DISCUSSION TITLE
installing on ext4 partitions

### DISCUSSION THREAD

**Discussion by mexsudo** [SpiralLinux/SpiralLinux-project#481] [Q&A]
I chose to install on ext4 partitions, all went fine.... until I updated.
default in GRUB is pointing to btrfs, and fails to boot.
So, I selected the previous kernel, and it's OK.
Update GRUB throws a btrfs error message.

Question: is this easy to repair or should I just reinstall using btrfs?

**Comment by geckolinux** [comment]
Hi there, did you have a previous Btrfs installation from another distro maybe? SpiralLinux doesn't require Btrfs or default to it if you install with another filesystem type. What installation mode did you use in SpiralLinux, "Erase Disk" or "Replace a partition" or?

**Comment by mexsudo** [comment]
no previous Btrfs systems... various with ext4 partitions, and a windows
I did a manual install, created new ext4 partitions.

**Reply by geckolinux** [reply]
I'm not sure what might have gone wrong, I've tested it to work (and update properly) on setups similar to yours. Maybe one of the other OSes messed up something? I guess you could re-install, please post a screenshot here of the manual partitioning screen.

**Comment by mexsudo** [comment]
update: I installed alongside using btfrs partitions
after update the New system is not in my GRUB menu.
running Debian I reinstalled and updated GRUB... still not in the menu

**Reply by geckolinux** [reply]
That's a know issue in Debian, with a workaround: https://github.com/SpiralLinux/SpiralLinux-project/issues/3

**Reply by mexsudo** [reply]
unfortunately I was unable to make that work.
Thanks, I will try again another day
